### PR TITLE
Used TTL index

### PR DESCRIPTION
**Pull request by romeOz**
See issue #16.

### CODE REVIEW

File: Session.php [hunk 1]
```diff
@@ -61,6 +61,9 @@ public function init()
     {
         parent::init();
         $this->db = Instance::ensure($this->db, Connection::className());
+        $this->db
+            ->getCollection($this->sessionCollection)
+            ->createIndex('expire', ['expireAfterSeconds' => 0]);
```
Comment (klimov-paul) [inline Session.php]: This is not appropriate.
`createIndex()` will be called at each application request process.

**Comment by romeOz** [comment]
Fix: "...  session doesn't active"

File: Session.php [hunk 2]
```diff
@@ -61,6 +61,11 @@ public function init()
     {
         parent::init();
         $this->db = Instance::ensure($this->db, Connection::className());
+        if (!$this->getIsActive()) {
+            $this->db
+                ->getCollection($this->sessionCollection)
+                ->createIndex('expire', ['expireAfterSeconds' => 0]);
```
Comment (klimov-paul) [inline Session.php]: This does not change much.
Index is not supposed to be created automatically. It should be created by migration script and not belong here.
That is why back at #16 I said this should be an option like `useTtl`, which can be enabled at will.

File: Session.php [hunk 3]
```diff
@@ -187,7 +192,7 @@ public function destroySession($id)
     public function gcSession($maxLifetime)
     {
         $this->db->getCollection($this->sessionCollection)
-            ->remove(['expire' => ['$lt' => time()]]);
+            ->remove(['expire' => ['$lt' => new \MongoDate()]]);
```
Comment (klimov-paul) [inline Session.php]: I don't get: if we rely on MongoDB internal mechanism for garbage collection, why manually do the same thing?

Comment (romeOz) [inline Session.php]: @klimov-paul, All right. When using the index, the garbage collector doesn't needed.

**Pull request closed without merging**